extensions/wikia/CommunityPage/templates/moderatorsModule.mustache

```diff
@@ -0,0 +1,21 @@
+{{#topModeratorsData}}
+    {{#topModeratorsModuleEnabled}}
+    <div class="module moderators-module">
```

Comment:
Per offline discussion, either we go full BEM or not. If we start with only a few classes, it'll introduce additional complexity. So, we agreed to use the prefixed notation but without introducing BEM